app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,9 @@
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
     <%= javascript_importmap_tags %>
+
+    <%= turbo_refreshes_with method: :morph, scroll: :preserve %>
```

Comment:
I'm experimenting with Turbo 8 via Phlex components and was surprised to see that the meta tags all use the `provide :head` construct instead of just emitting a tag.

If you want to make the tags accessible from outside of the content view, you might either eliminate the `provide :head` construct or keep that and expand out the methods like this:

```
  # Pages that are more likely than not to be a cache miss can skip turbo cache to avoid visual jitter.
  # Cannot be used along with +turbo_exempts_page_from_preview+.
  def turbo_exempts_page_from_cache
    provide :head, turbo_exempts_page_from_cache_tag
  end

  def turbo_exempts_page_from_cache_tag
    tag.meta(name: "turbo-cache-control", content: "no-cache")
  end
```

I think the reason you want to keep the `provide :head` tag is because the meta tags within each view file needs to put the meta tag in the head? The approach I provided above should serve both needs.

Reply:
Here's what I ended up doing in my Phlex layout:

```ruby
        # Turbo 8 has the helper method `turbo_refreshes_with method: :morph, scroll: :preserve` that
        # inserts the content in the `head` tag. Since Phlex doesn't work that way, we'll just put the tags here.
        meta(name: "turbo-refresh-method", content: "morph")
        meta(name: "turbo-refresh-scroll", content: "preserve")
```

Not a huge deal, but it would be nice to go through the logic that checks for argument validity if the tags are broken apart.

Reply:
I opened a PR at https://github.com/hotwired/turbo-rails/pull/542 that decouples tag generation from the way it's served. We can continue this discussion over there.